internal/load/models.go

```diff
@@ -164,6 +169,16 @@ func createNamespaceAndName(namingCfg NamingConfig, buildCfg BuildConfig, idx in
 	return namingCfg.Namespace, fmt.Sprintf("%s-%s-%d", namingCfg.Prefix, buildCfg.ClusterBuildStrategy, idx)
 }
 
+func createBuildAnnotations(buildCfg BuildConfig) map[string]string {
+	if buildCfg.SkipVerifySourceRepository {
+		annotations := make(map[string]string)
+		annotations[buildv1alpha.AnnotationBuildVerifyRepository] = "false"
+		return annotations
```

Comment:
Maybe a bit more compact?
```suggestion
		return map[string]string{
			buildv1alpha.AnnotationBuildVerifyRepository: "false",
		}
```

Reply:
Thank you. I was looking for how to use a variable value as key during map initialization, but had not found it. :-)

Reply:
Done.